packages/web/components/complex/paginated-table.tsx

```diff
@@ -0,0 +1,134 @@
+import { flexRender, Row, Table } from "@tanstack/react-table";
+import { useVirtualizer } from "@tanstack/react-virtual";
+import Image from "next/image";
+import Link from "next/link";
+import { useRef } from "react";
+import { IS_FRONTIER } from "../../config";
+import { Pool } from "./all-pools-table-set";
+
+const SIZE = 80;
+
+type Props = {
+  paginate: () => void;
```

Comment:
I don't see this actually get called, does this table actually paginate?

Reply:
Does not paginate, but instead calls `fetchRemainingPools`. I think I mentioned to you earlier the paginate function call isn't consistent, but `fetchRemainingPools` works well so I thought we were going with that. I do need to re-enable it, which I'm working on.

Reply:
@jonator Should I rename the function to better reflect that we are fetching remaining pools instead of paginating? Might want to keep it in case other use cases actually use pagination.

Reply:
Ah no that makes sense. When are remaining pools fetched when the user is not interacting with the sort/search controls? It should load remaining when they scroll to bottom of viewbox.

Reply:
Also for the container ref.. make sure it's a ref to the container actually scrolling the *entire* page. Not the immediate container of the table. What matters is `react-table` has a ref to the element handling the scrolling.. otherwise it doesn't know when the page was scrolled to bottom.

Reply:
_Also_ make sure that `containerRef` _itself_ is optional, then only use `getScrollElement` when needed